api/src/main/java/org/openmrs/api/db/ConditionDAO.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.db;
+
+import java.util.List;
+
+import org.openmrs.Condition;
+import org.openmrs.Patient;
+
+/**
+ * This interface defines database methods for condition objects.
+ *
+ * @since 2.2
+ */
+public interface ConditionDAO {
+
+	/**
+	 * Saves a new condition or updates an existing one if any.
+	 *
+	 * @param condition the condition to save or update.
+	 * @return the updated or saved condition.
+	 */
+	Condition saveCondition(Condition condition);
+
+	/**
+	 * Get the condition attached to the specified UUID.
```

Comment:
Can we change the wording from Get to Gets?

Reply:
@dkayiwa Should this apply to all the other "get" instances as well?